lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -19,12 +19,13 @@
           </p>
         </div>
         <div ng-if="main.selection.hasImage()">
-          <div class="step-selection-text">
+          <div class="step-selection-text"
+            uib-popover="{{ main.selection.getImagePath() }}"
+            popover-title="{{ main.selection.getImagePath() | basename }} - {{ main.selection.getImageSize() | gigabyte | number:1 }}GB"
+            popover-trigger="{'mouseenter': 'none', 'outsideClick': 'outsideClick'}">
```

Comment:
It seems like `$uibTooltipProvider.setTriggers` only affects tooltips and not popovers, which is odd.